StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetError.swift

```diff
@@ -129,6 +130,8 @@ extension PaymentSheetError: CustomDebugStringConvertible {
                 return message
             case .embeddedPaymentElementAlreadyConfirmedIntent:
                 return "This instance of EmbeddedPaymentElement has already confirmed an intent successfully. Create a new instance of EmbeddedPaymentElement to confirm a new intent."
+            case .integrationError(debugDescription: let debugDescription):
```

Comment:
I could modify this to `case integrationError(nonPIIDebugDescription)` to assuage FUD, we've done that elsewhere